app/src/ui/pages/TradeDetail.vue

```diff
@@ -13,26 +14,33 @@ import { usePriceStore } from '~/stores/price'
 import { microDenomToDenom } from '~/utils/denom'
 
 const client = useClientStore()
+const { userWallet } = storeToRefs(client)
 const priceStore = usePriceStore()
 const tradeInfo = ref()
-
+const trade = ref()
```

Comment:
Is this variable being used?

Reply:
It was being used to trick the compiler (ESLint) into not complaining from the bind on the line 100, however the bind was not necessary and removed together with the variable.